src/js/common/helpers/login-helpers.js

```diff
@@ -4,28 +4,13 @@ import environment from './environment.js';
 import { updateLoggedInStatus } from '../../login/actions';
 import { updateProfileField } from '../../user-profile/actions';
 
-export function handleLogin() {
-  this.serverRequest = fetch(`${environment.API_URL}/v0/sessions/new?level=1`, {
-    method: 'GET',
-  }).then(response => {
-    return response.json();
-  }).then(json => {
-    const myVerifyUrl = json.authenticate_via_get;
-    const receiver = window.open(myVerifyUrl, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
-    receiver.focus();
-  });
-}
-
-export function handleVerify() {
-  this.serverRequest = fetch(`${environment.API_URL}/v0/sessions/new?level=3`, {
-    method: 'GET',
-  }).then(response => {
-    return response.json();
-  }).then(json => {
-    const myVerifyUrl = json.authenticate_via_get;
-    const receiver = window.open(myVerifyUrl, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
+export function handleVerify(verifyUrl) {
+  window.dataLayer.push({ event: 'verify-link-clicked' });
+  if (verifyUrl) {
```

Comment:
Yeah, what Anne said. Basically, we need to decouple the user action (button/link click) from the URL fetch since the fetch callback can't open the window once the URL response is returned since most browsers will block it.  This is on the list of issues to file as follow-up to this PR.